src/community/params-extractor/src/main/java/org/geoserver/params/extractor/Filter.java

```diff
@@ -33,34 +47,52 @@ public int getPriority() {
     }
 
     @Override
-    public void init(FilterConfig filterConfig) throws ServletException {}
+    public void init(FilterConfig filterConfig) throws ServletException {
```

Comment:
So this method does not get called when the bean is just picked up from the spring context, but only when using it as a servlet filter, right? I'd clarify this.

Reply:
yes, should I write a comment on the method?

Reply:
Yes please :-)

Reply:
done